src/lava/lib/optimization/solvers/generic/nebm/models.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+
+from lava.lib.optimization.solvers.generic.nebm.process import NEBM
+from lava.magma.core.decorator import implements, requires, tag
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.model.py.ports import PyInPort, PyOutPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+
+
+@implements(proc=NEBM, protocol=LoihiProtocol)
+@requires(CPU)
+@tag('fixed_pt')
+class NEBMPyModel(PyLoihiProcessModel):
+    """Fixed point implementation of Stochastic Constraint Integrate and
```

Comment:
Change docstring:
Fixed point implementation of Boltzmann neuron for solving QUBO problems